.gitignore

```diff
@@ -9,3 +9,8 @@
 /build
 /captures
 .externalNativeBuild
+keystore.properties
+key_store
+app/release/
+api_key.txt
+Amazon-Android-SDKs/
```

Comment:
Amazon mobile sdk for now we only need lwa .jar

Reply:
coo broo